tests/actions/drug.test.js

```diff
@@ -1,21 +1,9 @@
-import expect from 'expect';
-import configureMockStore from 'redux-mock-store';
-import thunk from 'redux-thunk';
-import moxios from 'moxios';
-
 import { searchDrug } from '../../app/js/actions/drug';
 
 import * as actionTypes from '../../app/js/actions/actionTypes';
```

Comment:
@zeze111 , should not this also be importing specific modules?

Reply:
right, thanks